src/static-build/pages/index/index.tsx

```diff
@@ -27,7 +28,25 @@ const Index: FunctionalComponent<Props> = () => (
       <title>Squoosh</title>
       <meta
         name="description"
-        content="Compress and compare images with different codecs, right in your browser"
+        content="Squoosh is the ultimate image optimizer that allows you to compress and compare images with different codecs in your browser."
+      />
+      <meta
+        name="keywords"
+        content="best image optimizer, png to webp, online image compressor, image optimizer, online image optimizer, compress image, jpg to webp, jpg to avif, shrink image size"
+      />
+      <meta property="og:title" content="Squoosh" />
+      <meta property="og:type" content="website" />
+      <meta property="og:image" content={ogImage} />
+      <meta property="og:image:type" content="image/png" />
+      <meta property="og:image:width" content="500" />
+      <meta property="og:image:height" content="500" />
+      <meta
+        property="og:image:alt"
+        content="A cartoon of a hand squeezing an image file on a dark background."
```

Comment:
For some reason, the alt isn’t actually recognized by Facebook?

<details>

<img width="808" alt="Screen Shot 2021-07-19 at 12 21 06" src="https://user-images.githubusercontent.com/234957/126152717-a8973854-00f7-450d-b927-967db0482e01.png">


</details>

Reply:
Interesting, let me research this a bit more

Reply:
Sadly seems like a relatively undocumented issue. (also many devs forget to put an alt) Here's what I found:

- The Facebook crawler has a bug, in 2020 users discovered that Facebook's crawler was unable to decipher between `og:image` and `og:image:secure`. I think it is worth trying again with `secure` and let me know if that helps. More info: https://stackoverflow.com/questions/8855361/fb-opengraph-ogimage-not-pulling-images-possibly-https